eng/pipelines/common/controlgallery-android.yml

```diff
@@ -0,0 +1,87 @@
+steps:
+  - checkout: self
+    clean: true
+
+  - task: xamops.azdevex.provisionator-task.provisionator@1
+    displayName: 'Provisionator'
+    condition: eq(variables['provisioning'], 'true')
+    inputs:
+      provisioning_script: $(provisionator.path)
+      provisioning_extra_args: $(provisionator.extraArguments)
+  
+  - task: Bash@3
+    displayName: 'Cake Provision'
+    condition: eq(variables['provisioningCake'], 'true')
+    inputs:
+      targetType: 'filePath'
+      filePath: 'build.sh'
+      arguments: --target provision --TeamProject="$(System.TeamProject)"
+
+  - task: UseDotNet@2
+    displayName: 'Install .net core $(DOTNET_VERSION)'
+    condition: ne(variables['DOTNET_VERSION'], '')
+    inputs:
+      version: $(DOTNET_VERSION)
+      packageType: 'sdk'
+  
+  # - script: '/bin/bash -c "sudo $AGENT_HOMEDIRECTORY/scripts/select-xamarin-sdk.sh ${{ parameters.monoVersion }}"'
+  #   displayName: 'Select MONO ${{ parameters.monoVersion }}'
+  
+  - task: NuGetCommand@2
+    displayName: 'NuGet restore Xamarin.Forms'
+    inputs:
+      command: restore
+      restoreSolution: $(SolutionFile)
+      feedsToUse: config
+      nugetConfigPath: 'eng/DevopsNuget.config'
```

Comment:
Do we need these? I think the cake script handles these restores
https://github.com/xamarin/Xamarin.Forms/blob/main-handler/build.cake#L1145